views/chooseAll-answer.pug

```diff
@@ -0,0 +1,45 @@
+if answerForm
+  .form-group.col.s12
+    - for (var index = 1; index <= 4; index++)
+      .row
+        .input-field.col.s1
+          input(type='checkbox', name='checkButton', id='mcans'+index)
+          label(for='mcans'+index)
```

Comment:
the id `mcans` is still present, this is not an appropriate name, this applies to the rest of the file

Reply:
done